.rubocop.yml

```diff
@@ -1,1892 +1,87 @@
-  Enabled: false
-
-# Supports --auto-correct
-Lint/UnneededCopDisableDirective:
-  Description: 'Checks for rubocop:disable comments that can be removed. Note: this
-    cop is not disabled when disabling all cops. It must be explicitly disabled.'
-  Enabled: false
+Layout/EmptyLinesAroundAccessModifier:
+  EnforcedStyle: only_before
 
-Lint/UnreachableCode:
-  Description: Unreachable code.
-  Enabled: false
+Layout/LineLength:
+  Max: 125
+  AutoCorrect: true
 
-# Supports --auto-correct
-Lint/UnusedBlockArgument:
-  Description: Checks for unused block arguments.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars
-  Enabled: false
-  IgnoreEmptyBlocks: true
+Lint/SuppressedException:
+  Exclude:
+    - "test/**/*"
 
-# Supports --auto-correct
+# Because we use NotImplementErrors, we should
+# not care about this in exercises
 Lint/UnusedMethodArgument:
-  Description: Checks for unused method arguments.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars
-  Enabled: false
-  AllowUnusedKeywordArguments: false
-  IgnoreEmptyMethods: true
-
-Lint/UselessAccessModifier:
-  Description: Checks for useless access modifiers.
-  Enabled: false
-
-Lint/UselessAssignment:
-  Description: Checks for useless assignment to a local variable.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars
-  Enabled: false
-
-Lint/UselessComparison:
-  Description: Checks for comparison of something with itself.
-  Enabled: false
-
-Lint/UselessElseWithoutRescue:
-  Description: Checks for useless `else` in `begin..end` without `rescue`.
-  Enabled: false
-
-Lint/UselessSetterCall:
-  Description: Checks for useless setter call to a local variable.
-  Enabled: false
-
-Lint/Void:
-  Description: Possible use of operator/literal/variable in void context.
-  Enabled: false
-
-# Type 'Metrics' (9):
-Metrics/AbcSize:
-  Description: A calculated magnitude based on number of assignments, branches, and
-    conditions.
-  Reference: http://c2.com/cgi/wiki?AbcMetric
-  Enabled: false
-  Max: 15
+  Exclude:
+    - "exercises/**/*"
 
 Metrics/BlockLength:
-  Enabled: false
-  Max: 27
-
-Metrics/BlockNesting:
-  Description: Avoid excessive block nesting
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count
-  Enabled: false
-  Max: 3
-
-Metrics/ClassLength:
-  Description: Avoid classes longer than 100 lines of code.
-  Enabled: false
-  CountComments: false
-  Max: 100
-
-Metrics/CyclomaticComplexity:
-  Description: A complexity metric that is strongly correlated to the number of test
-    cases needed to validate a method.
-  Enabled: false
-  Max: 6
-
-Metrics/LineLength:
-  Description: Limit lines to 80 characters.
-  StyleGuide: https://github.com/bbatsov/ruby-style-guide#80-character-limits
-  Enabled: false
-  Max: 80
-  AllowHeredoc: true
-  AllowURI: true
-  URISchemes:
-  - http
-  - https
+  Exclude:
+    - "test/**/*"
```

Comment:
If we exclude metrics for tests, and pay attention to metrics in others, then we probably don't need to `--except Metrics` when running Rubocop in CI.